tools/gfx/render.cpp

```diff
@@ -233,6 +236,40 @@ extern "C"
         return SLANG_OK;
     }
 
+    SLANG_GFX_API SlangResult SLANG_MCALL gfxGetAdapters(DeviceType type, AdapterInfo* outAdapters, Size bufferSize, GfxCount* outAdapterCount)
```

Comment:
This two-call idiom is OK, but we've been moving to just return an allocated ISlangBlob to return buffers.